website_product_document_download_counter/tests/test_product_document_http.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/LGPL).
+from odoo.tests.common import HttpCase, tagged
+from odoo.tools import mute_logger
+
+from .product_document_test_utils import ProductDocumentTestUtils
+
+ODOO_LOGGER = "odoo"
+SQL_LOGGER = f"{ODOO_LOGGER}.sql_db"
+HTTP_LOGGER = f"{ODOO_LOGGER}.http"
+BASE_LOGGER = f"{ODOO_LOGGER}.addons.website_product_document_download_counter"
+MODEL_LOGGER = f"{BASE_LOGGER}.models.product_document"
+MAIN_LOGGER = f"{BASE_LOGGER}.controllers.main"
+
+
+@tagged("post_install", "-at_install")
+class TestProductDocumentDownloadCounterHttp(HttpCase, ProductDocumentTestUtils):
+    def setUp(self):
+        super().setUp()
+        # Create a new admin user for HTTP tests
+        admin_group = self.env.ref("base.group_system")
+        self.admin_user = (
+            self.env["res.users"]
+            .with_context(**{"no_reset_password": True})
```

Comment:
You should simply import [BaseCommon](https://github.com/odoo/odoo/blob/2f6eb4bfbafa73d10dd18fa588b4a5a2c8930dd6/odoo/addons/base/tests/common.py#L20) class in order to have everything already patch.